metricshub-web/react/src/components/navbar/status/status-details-menu.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import {
+	IconButton,
+	Menu,
+	Box,
+	CircularProgress,
+	Tooltip,
+	Typography,
+	List,
+	ListItem,
+	ListItemText,
+	Divider,
+} from "@mui/material";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import { useAppSelector } from "../../../hooks/store";
+
+// key prettifier
+function prettifyKey(key = "") {
```

Comment:
This will probably need to be moved in a separate convenience service, as we will need the same function in other places.